lib/fauna/client.rb

```diff
@@ -100,6 +100,16 @@ def query(expression)
       post('', expression)
     end
 
+    ##
+    # Ping the server.
```

Comment:
Maybe something more like "Ping FaunaDB" or something? 'ping the server' is a bit vague.